script.config

```diff
@@ -3,3 +3,4 @@ localPath = C:\Projects\
 editor = code
 username = 
 password = 
+twoFacAuth = false
```

Comment:
@MisterWhopper is this needed?

Reply:
We are going to need a way to tell if we need to do 2FA, but I think I can determine that at runtime. If not then I will add it in that PR

Reply:
my line of thought was that we’d have the tfa code in the main but only run it if this is true

Reply:
I'd like it to do a run-time check if that's possible

Reply:
Yeah, but when the 2FA is needed it returns a really specific error message which we can detect. It's something like "Must specify two factor authentication one time password" or something like that

Reply:
I see